src/gui/mainwindow.cpp

```diff
@@ -187,6 +189,16 @@ void MainWindow::neovimMaximized(bool set)
 	}
 }
 
+void MainWindow::neovimFrameless(bool isFrameless)
+{
+	setWindowFlag(Qt::FramelessWindowHint, isFrameless);
+	// DD: Need call show o make the widget visible again.
```

Comment:
Pure curiosity... What does "DD:" stand for?

I've never run across this acronym  before. Totally fine to keep as-is!

Reply:
DD is my initials (Dmitry Danilov), sorry for the confusion.